src/components/commanderOverview/CommanderDetails.tsx

```diff
@@ -42,6 +48,18 @@ export const CommanderDetails = React.memo(function CommanderDetails() {
         return <Loading text="" />;
     }
 
+    let matchesArray: Match[] = matches;
+    if (searchInput.length > 0 && matches) {
+        matchesArray = matches.filter((match: Match) => {
+            for (let i = 0; i < match.players.length; i++) {
+                if (match.players[i].name.toLowerCase().includes(searchInput.toLowerCase())) {

```

Comment:
can we do a filter by commander OR player here

Reply:
Done